### PR TITLE
Connect classroom backend

### PR DESCRIPTION
**Pull request by dhirpatel133**
- connecting with backend to make list of student and teacher classes
- created toggle functionality to show two different types of lists
- user is able to join a class
- updated backend query to retrieve list of student classes

### CODE REVIEW

File: backend/src/main/kotlin/clarity/backend/controllers/SpeechAnalysis.kt [hunk 1]
```diff
@@ -32,8 +32,11 @@ class SpeechAnalysis {
             }
             speechRegion = dotenv["SPEECH_REGION"]?: ""
             speechKey = dotenv["SPEECH_KEY"]?: ""
+            println("speechKey is: ${speechKey}")
```
Comment (ad-world) [inline backend/src/main/kotlin/clarity/backend/controllers/SpeechAnalysis.kt:35]: can u remove these prints

File: backend/src/main/kotlin/clarity/backend/entity/ClassroomEntity.kt [hunk 2]
```diff
@@ -104,13 +104,15 @@ class ClassroomEntity() {
         try {
             val statement = db!!.createStatement()
             val selectStatement = """
-                SELECT * FROM ClassroomStudents WHERE user_id = $userId
+                SELECT * FROM ClassroomStudents INNER JOIN Classroom
```
Comment (ad-world) [inline backend/src/main/kotlin/clarity/backend/entity/ClassroomEntity.kt:107]: What does this updates query do

**Review by ad-world — CHANGES_REQUESTED** [review]
left some comments, looks good otherwise

File: backend/src/main/kotlin/clarity/backend/entity/ClassroomEntity.kt (hunk 2 above)
Comment (dhirpatel133) [reply backend/src/main/kotlin/clarity/backend/entity/ClassroomEntity.kt:107]: it fetches the list of classes that the user does not own (i.e. the user is not the teacher of that class), only the classes that the students are part of.

File: backend/src/main/kotlin/clarity/backend/controllers/SpeechAnalysis.kt (hunk 1 above)
Comment (dhirpatel133) [reply backend/src/main/kotlin/clarity/backend/controllers/SpeechAnalysis.kt]: done

**Review by talin2412 — APPROVED** [review]

**Pull request merged**